### DISCUSSION TITLE
Mesh points are limited to 9x9 matrix

### DISCUSSION THREAD

**Discussion by jxer** [mriscoc/Ender3V2S1#166] [Q&A] [answered]
I am coming from jyers, I was able to do a 15x15 mesh, which worked out to about 145 points. I am seeing this only goes to 9x9, is there a way to adjust it higher?

**Comment by mriscoc** [comment, marked as answer]
No, this firmware limits the mesh size to a 9x9 matrix, that is enough for beds about 450 x 450 mm^2. You can change that limit editing the firmware but some text formatting problems could arise.

**Reply by jxer** [reply]
may I ask why jyers was able to do it without issue?

**Reply by mriscoc** [reply]
Just a matter of preference, I prefer that practical limit, more points are more a placebo effect for small beds and that limit allows to save up to 6 different mesh slots.

**Comment by Nbrooks639** [comment]
there is no need for 15x15 mesh which does not equal 145 points,
